sql/src/main/java/io/crate/planner/v2/ESGetConsumer.java

```diff
@@ -78,6 +79,11 @@ public PlannedAnalyzedRelation visitSelectAnalyzedStatement(SelectAnalyzedStatem
             WhereClauseAnalyzer whereClauseAnalyzer = new WhereClauseAnalyzer(analysisMetaData, tableRelation);
             WhereClauseContext whereClauseContext = whereClauseAnalyzer.analyze(statement.whereClause());
 
+            if(whereClauseContext.whereClause().version().isPresent()){
```

Comment:
GetRequest would actually support setting a version, not sure if we want to support that now ?